iaso/api/common.py

```diff
@@ -413,3 +413,30 @@ class IsAdminOrSuperUser(permissions.BasePermission):
 
     def has_permission(self, request, view):
         return bool(request.user and request.user.is_staff) or (request.user and request.user.is_superuser)
+
+
+class GenericReadWritePerm(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if request.method == "GET":
```

Comment:
Do we want `HEAD` and `OPTIONS` to be forbidden?

Otherwise [this could allow](https://github.com/encode/django-rest-framework/blob/0abb84fa3955956fc6156cc2c51222cd4ceae783/rest_framework/permissions.py#L8C17-L8C41) `GET` + `HEAD` + `OPTIONS`:

```suggestion
        if request.method in permissions.SAFE_METHODS:
```

We could have `can_read` and `can_write`.